ruby-challenges.rb

```diff
@@ -11,17 +20,71 @@
 # Expected output: ['tea', 'water', 'soda water']
 
 
+def letter_inside(array, letter)
+   array.select do |string| 
+   string.include?(letter)
+    end
+end
+    p letter_inside(beverages_array,letter_o)
+# Output:["coffee", "soda water"]
+    p letter_inside(beverages_array,letter_t)
+# Output: ["tea", "water", "soda water"]
+
+
 # -------------------2) Create a method that takes in a hash and returns one array with all the hash values at their own index and in alphabetical order. No nested arrays. Use the test variable provided.
 # HINT: Google 'ruby get rid of nested arrays'
 
+# Pseudo
+
+# Declare a method called no_nest with the parameter hash
+# Set a new variable called values
+# Set values equal to the hash.values.flatten 
+# The .flatten method removes the {}, and hash.values looks at only the values, without the keys
+# Then take values and use the sort method with the bang operator, this will store the sorted results in an array
+#  print no_nest with the arguement (us_states)
+
 us_states = { northwest: ['Washington', 'Oregon', 'Idaho'], southwest: ['California', 'Arizona', 'Nevada'], notheast: ['Maine', 'New Hampshire', 'Rhode Island'] }
+
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
 
+def no_nest hash
+  values = hash.values.flatten
+  values.sort! 
+end
```

Comment:
Awesome! No need to make `values`, you can just slap `.sort` to the end of `.flatten`!